lib/pf/services/manager/radiusd_child.pm

```diff
@@ -408,6 +441,50 @@ sub generate_radiusd_sqlconf {
    parse_template( \%tags, "$conf_dir/radiusd/sql.conf", "$install_dir/raddb/mods-enabled/sql" );
 }
 
+=head2 generate_radiusd_ldap
+Generates the ldap_packetfence configuration file
+=cut
+
+sub generate_radiusd_ldap {
+   my ($self, $tt) = @_;
+
+   my %tags;
+   $tags{'template'}    = "$conf_dir/radiusd/ldap_packetfence.conf";
+   $tags{'install_dir'} = $install_dir;
+   foreach my $ldap (keys %ConfigAuthenticationLdap) {
+      $tags{'servers'} .= <<"EOT";
+
+ldap $ldap {
+    server          = "$ConfigAuthenticationLdap{$ldap}->{host}"
+    port            = "$ConfigAuthenticationLdap{$ldap}->{port}"
+    identity        = "$ConfigAuthenticationLdap{$ldap}->{binddn}"
+    password        = "$ConfigAuthenticationLdap{$ldap}->{password}"
+    base_dn         = "$ConfigAuthenticationLdap{$ldap}->{basedn}"
+    filter          = "(userPrincipalName=%{User-Name})"
```

Comment:
We should quote the username to avoid ldap injection.
"(userPrincipalName=%{ldapquote:&User-Name})"

Reply:
ldapquote

Reply:
I tested it but after that it doesn't work, so i revert.

Reply:
Ok